src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -45,8 +45,9 @@
  :price
  :<- [:prices]
  (fn [prices [_ fsym tsym]]
-   (get-in prices [fsym tsym :price])))
+   (get-in prices [fsym tsym])))
 
+;; NOTE(rasom): what's this?
```

Comment:
can we include a more descriptive note or else remove this ? imo it's just confusing for other developers otherwise